inception-ui-kb/src/main/java/de/tudarmstadt/ukp/inception/ui/kb/stmt/StatementGroupPanel.java

```diff
@@ -255,9 +255,12 @@ private void actionStatementChanged(AjaxRequestTarget target,
 
         private void actionAddValue(AjaxRequestTarget target) {
             // add a new prototype statement using this group's instance and its current property
-            KBStatement statementProto = new KBStatement();
-            statementProto.setInstance(groupModel.getObject().getInstance());
-            statementProto.setProperty(groupModel.getObject().getProperty());
+            KBStatement statementProto = new KBStatement(
+                groupModel.getObject().getInstance(),
+                groupModel.getObject().getProperty(),
+                null
```

Comment:
Why null? If the argument is optional, maybe introduce a new KBStatement constructor which does not have it.

Reply:
@reckart Because in the frontend of the knowledge base, the user first selects the property for a statement, at this moment, there is no value for this statement. I'll introduce a new KBStatement constructor.